src/Assets/Editor/AssetsPropertiesApplier.cs

```diff
@@ -143,6 +144,35 @@ private enum ComponentType
 							navMeshModifier.area = NavMesh.GetAreaFromName("Not Walkable");
 						}
 
+						break;
+					case ComponentType.Camera:
+						if (targetGameObject.GetComponent<Camera>() == null)
+						{
+							GameObject cameraObject = new GameObject();
+
+							Camera camera = cameraObject.AddComponent<Camera>();
+							camera.clearFlags = CameraClearFlags.Color;
+							camera.backgroundColor = new Color32(88, 85, 74, 255);
+							camera.orthographic = false;
+							switch (tag)
+							{
+								case "Plane":
+									cameraObject.tag = "PlaneCamera";
+									cameraObject.transform.position = new Vector3(-0.88f, 25.1f, -4.7f);
+									cameraObject.transform.rotation = Quaternion.Euler(70, 0, 0);
+									camera.fieldOfView = 47.1f;
```

Comment:
Magic number 47.1?

Reply:
A lot of random numbers here actually. You be the judge if this is future proof enough.

Reply:
Create a variable that holds the vector.

var positionForThisSpecificPurpose = new Vector3(-0.88f, 25.1f, -4.7f);
cameraObject.transform.position = positionForThisSpecificPurpose;